style.css

```diff
@@ -6,6 +6,7 @@
 @import "./component\ styles/Image.css";
 @import "./component\ styles/Input.css";
 @import "./component\ styles/Navigation.css";
+@import "./component\ styles/Text-utility.css";
```

Comment:
you can import by using the below approach , 

example:
@import url("./images.css");